Fix broken links

ShuheiUda opened the pull request:
Section title was changed in following commit.
https://github.com/MicrosoftDocs/azure-docs/commit/7cf20d512674cf8d8666dbb8c3db31061de9aa58#diff-7c3c39b0c68a082dad6b8ca904a7008dL72

azurecla commented:
Hi __@briglx__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@nitinme 
He's an MSFT person, so I accepted. Please check.

cjgronlund commented:
@briglx 
Thank you for your contribution.

ShuheiUda committed:
Fix broken links

Section title was changed in following commit.
https://github.com/MicrosoftDocs/azure-docs/commit/7cf20d512674cf8d8666dbb8c3db31061de9aa58#diff-7c3c39b0c68a082dad6b8ca904a7008dL72
